### PR TITLE
declarative labels for better backlog classification.

### PR DESCRIPTION
**Pull request by raulk**

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  name: area/builder-api
+  color: 1d76db
+- description: "Areas: Network reaper"
+  name: area/network-reaper
+  color: 1d76db
+- description: "Areas: Chaos Actor"
+  name: area/chaos-actor
+  color: 1d76db
+- description: "Areas: Docs"
+  name: area/chaos-actor
```
Comment (raulk) [inline .github/labels.yml:48]:
```suggestion
  name: area/docs
```

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+### Areas
+###
+- description: "Areas: Message-class vector"
+  name: area/message-vector
+  color: 1d76db
+- description: "Areas: Chain-class vector"
+  name: area/chain-vector
+  color: 1d76db
+- description: "Areas: Drivers"
+  name: area/driver
```
Comment (raulk) [inline .github/labels.yml:33]:
```suggestion
  name: area/drivers
```

File: .github/labels.yml [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+- description: "Effort: One Day."
+  name: effort/day
+  color: 64d5ca
+- description: "Effort: Multiple Days."
+  name: effort/days
+  color: 4dc0b5
+- description: "Effort: One Week."
+  name: effort/week
+  color: 38a89d
+- description: "Effort: Multiple Weeks."
```
Comment (raulk) [inline .github/labels.yml:128]:
```suggestion
- description: "Effort: Multiple weeks."
```

File: .github/labels.yml [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+- description: "Effort: One or Multiple Hours."
+  name: effort/hours
+  color: a0f0ed
+- description: "Effort: One Day."
+  name: effort/day
+  color: 64d5ca
+- description: "Effort: Multiple Days."
+  name: effort/days
+  color: 4dc0b5
+- description: "Effort: One Week."
```
Comment (raulk) [inline .github/labels.yml:125]:
```suggestion
- description: "Effort: One week."
```

File: .github/labels.yml [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+- description: "Effort: Minutes"
+  name: effort/minutes
+  color: e8fffe
+- description: "Effort: One or Multiple Hours."
+  name: effort/hours
+  color: a0f0ed
+- description: "Effort: One Day."
+  name: effort/day
+  color: 64d5ca
+- description: "Effort: Multiple Days."
```
Comment (raulk) [inline .github/labels.yml:122]:
```suggestion
- description: "Effort: Multiple days."
```

File: .github/labels.yml [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+###
+### Efforts
+###
+- description: "Effort: Minutes"
+  name: effort/minutes
+  color: e8fffe
+- description: "Effort: One or Multiple Hours."
+  name: effort/hours
+  color: a0f0ed
+- description: "Effort: One Day."
```
Comment (raulk) [inline .github/labels.yml:119]:
```suggestion
- description: "Effort: One day."
```

File: .github/labels.yml [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+  name: dif/expert
+  color: 2f365f
+
+###
+### Efforts
+###
+- description: "Effort: Minutes"
+  name: effort/minutes
+  color: e8fffe
+- description: "Effort: One or Multiple Hours."
```
Comment (raulk) [inline .github/labels.yml:116]:
```suggestion
- description: "Effort: One or multiple hours."
```

File: .github/labels.yml [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+  name: P2
+  color: fbca04
+- description: "P3: Might get fixed."
+  name: P3
+  color: 0e8a16
+
+###
+### Hints
+###
+- description: "Hint: Good First Issue"
```
Comment (raulk) [inline .github/labels.yml:188]:
```suggestion
- description: "Hint: Good first issue"
```

File: .github/labels.yml [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/good-first-issue
+  color: 0623cc
+- description: "Hint: Needs Contributor"
```
Comment (raulk) [inline .github/labels.yml:191]:
```suggestion
- description: "Hint: Needs contributor"
```

File: .github/labels.yml [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-contributor
+  color: 0623cc
+- description: "Hint: Needs Scoring"
```
Comment (raulk) [inline .github/labels.yml:194]:
```suggestion
- description: "Hint: Needs scoring"
```

File: .github/labels.yml [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-scoring
+  color: 0623cc
+- description: "Hint: Needs Decision"
```
Comment (raulk) [inline .github/labels.yml:197]:
```suggestion
- description: "Hint: Needs decision"
```

File: .github/labels.yml [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-decision
+  color: 0623cc
+- description: "Hint: Needs Triage"
```
Comment (raulk) [inline .github/labels.yml:200]:
```suggestion
- description: "Hint: Needs triage"
```

File: .github/labels.yml [hunk 13]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-triage
+  color: 0623cc
+- description: "Hint: Needs Analysis"
```
Comment (raulk) [inline .github/labels.yml:203]:
```suggestion
- description: "Hint: Needs analysis"
```

File: .github/labels.yml [hunk 14]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-analysis
+  color: 0623cc
+- description: "Hint: Needs Author Input"
```
Comment (raulk) [inline .github/labels.yml:206]:
```suggestion
- description: "Hint: Needs author input"
```

File: .github/labels.yml [hunk 15]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-author-input
+  color: 0623cc
+- description: "Hint: Needs Team Input"
```
Comment (raulk) [inline .github/labels.yml:209]:
```suggestion
- description: "Hint: Needs team input"
```

File: .github/labels.yml [hunk 16]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-team-input
+  color: 0623cc
+- description: "Hint: Needs Community Input"
```
Comment (raulk) [inline .github/labels.yml:212]:
```suggestion
- description: "Hint: Needs community input"
```

File: .github/labels.yml [hunk 17]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-community-input
+  color: 0623cc
+- description: "Hint: Needs Review"
```
Comment (raulk) [inline .github/labels.yml:215]:
```suggestion
- description: "Hint: Needs review"
```

File: .github/labels.yml [hunk 18]
```diff
@@ -0,0 +1,254 @@
...
+  name: hint/needs-review
+  color: 0623cc
+- description: "Hint: Needs Help"
```
Comment (raulk) [inline .github/labels.yml:218]:
```suggestion
- description: "Hint: Needs help"
```

**Review by raulk — COMMENTED** [review]
Normalise capitalisation.

**Comment by alanshaw** [comment]
🚀 nice!

**Pull request merged**